source/simulator/assemblers/newton_stokes.cc

```diff
@@ -438,21 +463,12 @@ namespace aspect
 
               if (enable_prescribed_dilation)
                 data.local_rhs(i) += (
-                                       // RHS of - (div u,q) = - (R,q)
                                        - pressure_scaling
-                                       * prescribed_dilation->dilation[q]
+                                       * (prescribed_dilation->dilation_rhs_term[q] -
+                                          prescribed_dilation->dilation_lhs_term[q] *
```

Comment:
why is this the difference between rhs and lhs term? Since we are assembling the rhs here, I would expect to only see the rhs side term. Please add a comment that explains this above line 464.